experiment_analysis/plotting.py

```diff
@@ -95,16 +98,34 @@ def get_timeseries(
                                                                     metric=f"{metric}_cuped",
                                                                     method=CappingMethod.WITH_CUPED,
                                                                     metric_raw=metric)
+        elif method == TimeseriesMethod.ISOTONIC:
```

Comment:
nit, not blocking: I wonder if there's a way to clean up these if statements to modularize the logic a bit better, because it looks like some of it is repeated across these different statements. For example, we need to do the same capping for `ISOTONIC_CAPPED` and `CUPED_CAPPED`, and need the same pre-period data across all of the CUPED & isotonic methods.